### ISSUE TITLE
Providing .graphql file with correct schema gives Loading schema.graphql gives `Query root type must be provided.` error

### ISSUE THREAD

**Issue by wesselvdv** [dotansimha/graphql-code-generator#812]
**Describe the bug**
Providing .graphql file with correct schema gives Loading schema.graphql gives `Query root type must be provided`. error

**To Reproduce**
1. Use the following schema:
```graphql
schema {
  query: QueryRoot
}

type QueryRoot {
  """Fetches an object given its ID"""
  node(
    """The ID of an object"""
    id: ID!
  ): Node
}
```
2. Use any document
3. Run the following command: `gql-gen --watch --schema './src/app/graphql/schema.graphql' --template graphql-codegen-apollo-angular-template --out './src/app/graphql/queries.ts' './src/**/*.graphql'`

**Expected behavior**
Expected schema to be loaded from .graphql file.

**Environment:**
 - OS: alpine
 - Codegen: 0.13.0
 - Node: 10.11

**Comment by kamilkisiela** [comment]
@wesselvdv I probably fixed it with #815 but maybe `schema { ... }` is not handled by the codegen, could you check `0.14.0-alpha.b04233ad`?

**Comment by kamilkisiela** [comment]
I tried to reproduce it as you can see in #816 but it seems everything's fine.

**Comment by kamilkisiela** [comment]
Actually it doesn't work but I fixed that in f1e5a0c05d18da32564a55e14db20de89220a0a2

**Comment by dotansimha** [comment]
Fixed in 0.14.0 🎉

**Comment by wesselvdv** [comment]
@dotansimha Using the example schema above still gives an error in 0.14.0:

`Unable to find the schema root matching query`

**Comment by furier** [comment]
Yeah me to having the same problem

```
$ gql-gen
 √ Parse configuration
 √ Load schema
 √ Load documents
 > Generate outputs
   > Generate src/app/shared/models/graphql/types.ts
     × Generate
       → Unable to find the schema root matching: query


     We found 1 errors

     Failed to generate src/app/shared/models/graphql/types.ts
Error: Unable to find the schema root matching: query
Something went wrong
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

**Comment by furier** [comment]
Sorry for nagging you, but I am very eager to test out the `graphql-codegen-apollo-angular` plugin to start using it in our angular application. :D Is this still *waiting-for-release* ?

**Comment by dotansimha** [comment]
@kamilkisiela is working on fixing it right now :) 
The issue is with the package that we are using for merging GraphQL schemas (`@graphql-modules/epoxy`).

**Comment by wesselvdv** [comment]
@dotansimha Also seems this issue is not necessarily related to using graphql(s) files. I am also experiencing this when using the previously working .json file.

**Comment by kamilkisiela** [comment]
It's related to the way schemas are being merged, a schema definition object is removed and it only treats `Query`, `Mutation` and `Subscription` as root types.

Fixed in #899

**Comment by dotansimha** [comment]
@wesselvdv @furier Can you please try with `0.15.0-alpha.7c1a7d2a` from npm? it includes the latest fixes that @kamilkisiela did .

**Comment by furier** [comment]
Have had meetings all day, but ill get right on it now. :)

**Comment by furier** [comment]
@dotansimha @kamilkisiela works like a charm when i upgraded all the packages to that version :)

**Comment by wesselvdv** [comment]
When will a new version be cut?

**Comment by dotansimha** [comment]
Fixed in 0.14.2 🎉
